tests/phpunit/integration/Core/Modules/Module_Sharing_SettingsTest.php

```diff
@@ -215,84 +215,133 @@ public function test_get_shared_roles() {
 		$this->assertEmpty( $this->settings->get_shared_roles( 'pagespeed-insights' ) );
 	}
 
+	private function grant_manage_options_permission() {
```

Comment:
Similar to my last comment, this method wouldn't be necessary I think if we split the test into more discrete checks rather than one long test case.

Reply:
Updated.